dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/datavalueset/DefaultDataValueSetService.java

```diff
@@ -832,6 +832,14 @@ private ImportSummary saveDataValueSet( ImportOptions importOptions, JobConfigur
                 ? context.getDataValueBatchHandler().findObject( internalValue )
                 : null;
 
+            // -----------------------------------------------------------------
+            // Preserve any existing created date unless overwritten by import
+            // -----------------------------------------------------------------
+            if ( existingValue != null && !dataValue.hasCreated() )
+            {
+                internalValue.setCreated( existingValue.getCreated() );
```

Comment:
Really makes me wonder if there are other fields we might "loose" same way that should be kept in a similar fashion.

Reply:
Very good question. With the data entry app, the primary key fields must stay the same, the "value", "storedby", and "lastupdated" fields change to reflect the update, and "deleted" gets reset.

The fields that do not change are created, comment, and followup. Should all three of these also be preserved when updating the data value in an import (if the import does not specify a value for them)? I'm not sure how to answer the question, but if so the logic can be added here.

Reply:
I've removed the `@Ignore` on `DataValueSetServiceIntegrationTest`.

Reply:
> The fields that do not change are created, comment, and followup.
Sounds to me that these should be preserved when not specified in import, @larshelge WDYT?

Reply:
Good spot. I think let's start with `created`, as created should be immutable and certainly should never change on update, even if specified in the payload.

Reply:
@larshelge Actually, if it's specified in the payload, it will change the existing `created` date. (See the code, and above: "if... the import does not specify a created date, keep the existing created date.")

Is this worth changing now, so the imported created date is ignored when updating values? What about for new values, should it also be ignored and just set to now?